tmd/Topology/analysis.py

```diff
@@ -347,7 +349,9 @@ def find_apical_point_distance_smoothed(ph, threshold=0.1):
         minimas = np.where(abs(der1) < threshold * np.max(abs(der1)))[0]
         minimas = minimas[der2[minimas] > 0]
         threshold *= 2.  # if threshold was too small, increase and retry
-    return bin_centers[minimas[0]]
+
+    last_bif = 0.9 * max([tmd_bar[1] for tmd_bar in ph])
```

Comment:
don't need a list, can just have `max(tmd_bar[1] for tmd_bar in ph)`

Reply:
good point!

Reply:
```suggestion
    last_bif = 0.9 * max(tmd_bar[1] for tmd_bar in ph)
```